resolwe_bio/processes/workflows/gene-fusion-calling-arriba.py

```diff
@@ -246,6 +253,9 @@ def run(self, inputs, outputs):
         if inputs.blacklist_file:
             arriba_input["blacklist_file"] = inputs.blacklist_file.id
 
+        if inputs.known_fusions:
+            arriba_input["known_fusions"] = inputs.known_fusions.id
```

Comment:
Should be `arriba_input[known_fusions_file]`. Consider making the variable name the same between the process and the workflow to avoid future confusions.

Reply:
If possible, add an example fusion file to tests to test the working of this option.